packs/journals/archetypes.json

```diff
@@ -946,7 +946,7 @@
+                "content": "<p>The warriors of Minata frequently contend with lost spirits of the Taumatan people that haunt their shattered lands to this day. These Minatan warriors, known as ghost eaters around Bonmu, travel the countless islands of Minata and offer their skills to exorcists and priests. When working in such a partnership, ghost eater's main task is to destroy a spirit in order to give their holy comrade time to learn about the individual soul and finally put it to rest.</p><p>Characters can gain access to this archetype from Ingdani after they retrieve the @UUID[Compendium.pf2e.equipment-srd.Item.Phoenix Necklace] from the temple of Irori.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Ghost Eater Dedication] <span style=\"float:right\">Feat 2</span></h2><p><strong>Prerequisites</strong> trained in Occultism or Religion</p><hr /><p>You have trained as a ghost eater, a hunter specialized in the destruction of spirits. You know how to channel your own spiritual energy to harm spirits. Your weapon and unarmed attack Strikes become magical. Against incorporeal creatures, they also gain the effects of a ghost touch property rune.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Reach Beyond] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.QMycbf2StuPcUbzO inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Disrupting Strikes] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Nm8n3urzpDqXni1i inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Cross the Threshold] <span style=\"float:right\">Feat 16</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.XXOc3MxbkEBMgFeT inline]</p>",
```

Comment:
```suggestion
                "content": "<p>The warriors of Minata frequently contend with lost spirits of the Taumatan people that haunt their shattered lands to this day. These Minatan warriors, known as ghost eaters around Bonmu, travel the countless islands of Minata and offer their skills to exorcists and priests. When working in such a partnership, ghost eater's main task is to destroy a spirit in order to give their holy comrade time to learn about the individual soul and finally put it to rest.</p><p>Characters can gain access to this archetype from Ingdani after they retrieve the @UUID[Compendium.pf2e.equipment-srd.Item.Phoenix Necklace] from the temple of Irori.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Ghost Eater Dedication] <span style=\"float:right\">Feat 2</span></h2><p><strong>Prerequisites</strong> trained in Occultism or Religion</p><hr /><p>You have trained as a ghost eater, a hunter specialized in the destruction of spirits. You know how to channel your own spiritual energy to harm spirits. Your weapon and unarmed attack Strikes become magical. Against incorporeal creatures, they also gain the effects of a <em>@UUID[Compendium.pf2e.equipment-srd.Item.Ghost Touch]</em> property rune.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Reach Beyond] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.QMycbf2StuPcUbzO inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Disrupting Strikes] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Nm8n3urzpDqXni1i inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Cross the Threshold] <span style=\"float:right\">Feat 16</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.XXOc3MxbkEBMgFeT inline]</p>",
```

Reply:
Consider also adding the link `<em>@UUID[Compendium.pf2e.equipment-srd.Item.Ghost Touch]</em>` to the Ghost Eater Dedication item itself.